themes/dark/assets/js/config/notifications.js

```diff
@@ -604,6 +604,28 @@ MEDUSA.config.notifications = function() { // eslint-disable-line max-lines
     // We have to call this function on dom ready to create the devices select
     getPushbulletDevices();
 
+    $('#testJoin').on('click', function() {
```

Comment:
This whole file is not used anymore. Im supprised its still here. Did you manually add your changes here. Ir was it build?

Reply:
I thought I had removed them from my side locally, but lo and behold they are still here, I will remove and push again.